### ISSUE TITLE
Error when saving post

### ISSUE THREAD

**Issue by maxleistner** [hingst/layotter#6]
## Setup
- Wordpress 4.3.1
- Layotter 1.5.4
- ACF Pro 5.3.1
## Details
- Layotter is NOT activated for post type blog post
- Layotter is ONLY activated for pages
## Steps to error
1. Edited Post Entry with ACF Fields
2. Entered Details into field
3. Click on "Update"
## Error

```
Notice: Undefined offset: 0 in /homepages/6/X/htdocs/vorschau/X/wp-content/plugins/layotter/lib/wp-post-meta-revisions.php on line 158

Warning: Cannot modify header information - headers already sent by (output started at /homepages/6/X/htdocs/vorschau/bellini/wp-content/plugins/layotter/lib/wp-post-meta-revisions.php:158) in /homepages/6/X/htdocs/vorschau/X/wp-admin/post.php on line 242

Warning: Cannot modify header information - headers already sent by (output started at /homepages/6/X/htdocs/vorschau/bellini/wp-content/plugins/layotter/lib/wp-post-meta-revisions.php:158) in /homepages/6/X/htdocs/vorschau/X/wp-includes/pluggable.php on line 1207
```

**Comment by hingst** [comment]
This issue from the bundled wp-post-meta-revisions plugin should be fixed in [1.5.5](https://github.com/hingst/layotter/releases/tag/v1.5.5) - please confirm.

**Comment by maxleistner** [comment]
Perfect!! Thanks a lot for this really quick reply and fix! Awesome! Works like a charm!
